lib/json.bash

```diff
@@ -75,7 +75,7 @@ JSON.keys() {
   JSON.object "$@" |
     cut -f1 |
     sed "s/^\///; s/\/.*//" |
-    sort -u
+    uniq
```

Comment:
Having just `uniq` here makes it look like it is assumed that the keys are already sorted. Because `uniq` only removes same values on adjacent lines. It would actually make more sense to either keep the `sort -u`, or drop it completely and just assume the user knows what he's doing. Since JSON is position independent, I'd vote to keep the `sort -u` in place and make it easier for the user.